kafka/src/main/java/org/axonframework/extensions/kafka/eventhandling/producer/KafkaSendingEventHandler.java

```diff
@@ -0,0 +1,22 @@
+package org.axonframework.extensions.kafka.eventhandling.producer;
+
+import org.axonframework.config.ProcessingGroup;
+import org.axonframework.eventhandling.EventHandler;
+import org.axonframework.eventhandling.EventMessage;
+
```

Comment:
I am not going to block this feature suggestion over it, as this repository isn't finalized any how.
Regardless I'd want to point out that any public classes/constructors/method/constants should have javadoc, thus including this object.